packages/medusa/src/api/store/orders/middlewares.ts

```diff
@@ -57,4 +71,15 @@ export const storeOrderRoutesMiddlewares: MiddlewareRoute[] = [
       ),
     ],
   },
+  {
+    method: ["POST"],
+    matcher: "/store/orders/:id/transfer/decline",
+    middlewares: [
+      validateAndTransformBody(StoreDeclineOrderTransferRequest),
```

Comment:
q: shouldn't this be auth'd as well?

Reply:
A guest customer should be able to accept / decline transfer of their order.
(a guest customer who is the original owner of the order needs to provide a token for this operation which is sent to them via e.g. email)

Reply:
I see, ok cool thanks!